sys/psa_crypto/include/psa_crypto_algorithm_dispatch.h

```diff
@@ -175,6 +175,21 @@ psa_status_t psa_algorithm_dispatch_verify_message( const psa_key_attributes_t *
  */
 psa_status_t psa_algorithm_dispatch_generate_key(   const psa_key_attributes_t *attributes,
                                                     psa_key_slot_t *slot);
+
+/**
+ * @brief   Dispatch the key import function to a specific backend.
+ *          See psa_import_key()
+ *
+ * @param attributes
+ * @param data
+ * @param data_length
+ * @param slot
+ * @param bits
+ * @return psa_status_t
```

Comment:
```suggestion
* @return  @ref PSA_SUCCESS
*          @ref PSA_ERROR_INVALID_ARGUMENT
 ...
```
(Maybe get's a bit long here to list all return values? Not sure if it is "better".
Just checked some established psa headers. They also do the `@return psa_status_t`. :thinking:
Feel free to dismiss this comment.)

Reply:
There are two different kinds of header files for PSA crypto right now: The "public" one that is supposed to be used by applications and other modules using the API (`psa/crypto.h` found in `sys/include/psa_crypto`) and several ones that are only used internally by the PSA crypto implementation in RIOT (found in `sys/psa_crypto/include`).

The latter ones for now contain much less thorough documentation than the former one. Would be nice to change that, but maybe not in scope of this PR?

Reply:
It would probably be nice to have more documentation. I was not very motivated to do that, because many parameters and return values are just the same as for the public functions, but of course that was just me being lazy.

I do agree that this is not in scope for this PR, though.

Reply:
I'd keep it as is since we agreed it is out of scope for this PR.